Integrations/ThinkstCanary/ThinkstCanary.py

```diff
@@ -0,0 +1,429 @@
+import demistomock as demisto
+from CommonServerPython import *
+from CommonServerUserPython import *
+
+''' IMPORTS '''
+import requests
+
+# disable insecure warnings
+requests.packages.urllib3.disable_warnings()
+
+# remove proxy if not set to true in params
+if not demisto.params().get('proxy', False):
+    del os.environ['HTTP_PROXY']
+    del os.environ['HTTPS_PROXY']
+    del os.environ['http_proxy']
+    del os.environ['https_proxy']
+
+''' GLOBALS'''
+
+SERVER = demisto.params().get('server').rstrip('/') + '/api/v1/'
+AUTH_TOKEN = demisto.params().get('auth_token')
+VERIFY_CERTIFICATE = not demisto.params().get('insecure', True)
+FETCH_DELTA = int(demisto.params().get('fetchDelta', 24))
+RELEVANT_DEVICE_ENTRIES = {
+    'description': 'Description',
+    'id': 'ID',
+    'ip_address': 'Address',
+    'last_seen': 'LastSeen',
+    'live': 'Status',
+    'location': 'Location',
+    'name': 'Name',
+    'updated_std': 'LastUpdated',
+    'version': 'Version'
+}
+RELEVANT_TOKEN_ENTRIES = {
+    'canarytoken': 'CanaryToken',
+    'created_printable': 'CreatedTime',
+    'enabled': 'Enabled',
+    'kind': 'Kind',
+    'triggered_count': 'Triggered',
+    'doc_name': 'DocName',
+    'url': 'TokenURL'
+}
+DEF_PARAMS = {
+    'auth_token': AUTH_TOKEN,
+}
+'''HELPER FUNCTIONS'''
+
+
+def http_request(method, url, params=None):
+    """
+    HTTP request helper function
+    """
+    if params is None:
+        params = DEF_PARAMS
+    res = requests.request(
+        method=method,
+        url=url,
+        params=params,
+        verify=VERIFY_CERTIFICATE
+    )
+
+    if not res.ok:
+        demisto.debug(res.text)
+        return_error('Could not execute the request')
+
+    try:
+        res_json = res.json()
+        return res_json
+    except Exception as ex:
+        demisto.debug(str(ex))
+        return_error(str(ex))
+
+
+def get_alerts(last_fetch=None):
+    """
+    Retrieve all unacknowledged alerts from Canary Tools
+    :param last_fetch: Last fetch incidents time
+    """
+
+    if last_fetch:
+        params = {
+            'auth_token': AUTH_TOKEN,
```

Comment:
why are we passing the `auth_token` in every request function as part of params? just add it in the http_request